js/animate_team.js

```diff
@@ -0,0 +1,20 @@
+var animate = anime({
+    targets: '.team__clip-path-container',
+    translateY: [
+        {value: -2000, duration: 200},
+        {value: 0, duration: 200}
+    ],
+    opacity: [
+        {value: 0, duration: 400},
+        {value: 1, duration: 200}
+    ],
+    delay: 1000,
```

Comment:
W przypadku pozostawienia tylko przezroczystości, rozważ zmniejszenie opóźnienia do 500 albo i jeszcze mniej. Sekunda to dużo czasu i można już przeskrolować na dół zanim się zacznie animacja.

Reply:
Ja mimo wszystko zostawiłabym 1000. Efekt przy takim opóźnieniu jest jakiś taki pełniejszy:)

Reply:
zmniejszylem na 500 dodalem jedno "mrugnięcie"